components/Button.vue

```diff
@@ -0,0 +1,30 @@
+<template>
```

Comment:
Where is this button component used?

Reply:
Not in use. Thanks for spotting this. It is deleted now.